src/lwc/gsChecklist/gsChecklist.js

```diff
@@ -0,0 +1,64 @@
+import { LightningElement, api, track } from 'lwc';
+import gsChecklistArrowRightAlterText from '@salesforce/label/c.gsChecklistArrowRightAlterText';
+import gsChecklistArrowDownAlterText from '@salesforce/label/c.gsChecklistArrowDownAlterText';
+
+export default class gsChecklist extends LightningElement {
+    
+    /**
+    * Group of step in a checklist
+    * @type      Checklist
+    */
+    @api group;
+    @track open = false;
+
+    /**
+    * Return if display the checklist item step
+    * @return      Boolean
+    * @see         Boolean
+    */
+    get displaySteps() {
+        return this.open;
+    }
+
+    /**
+    * Return css class to display info section
+    * @return      String
+    * @see         Css class
+    */
+    get infoClass() {
+        const classNames = ['slds-media', 'info'];
+        if(this.displaySteps) {
+            classNames.push('open');
+        }
+        return classNames.join(' '); 
+    }
+
+    /**
+    * Click event handler to display or not info section
+    * @return      void
+    * @see         Action
+    */
+    onClickOpenOrClose(event) {
+        this.open = !this.open;
+    }
+    
+    get expandedId() {
+        return `section-${this.group ? this.group.id : '0'}`;
```

Comment:
It looks like the quotes here are back-ticks (`) instead of single quotes ('). There should be multiple ways to escape or use other quote options. Either way, back-ticks should not be used.
```suggestion
        return 'section-${this.group ? this.group.id : "0"}';
```

or 

```suggestion
        return "section-${this.group ? this.group.id : '0'}";
```

Reply:
@force2b 
The **`** not have the same functionality in Js like **'** or **"** ,  I use [Template Strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
Thanks for the explanation. Didn't realize that back-ticks are now acceptable. I stand corrected, so feel free to ignore these two comments.